Podfile

```diff
@@ -153,7 +153,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.47.0-alpha2'
+    gutenberg :commit => '7f75b334f402ff36ac154274b728c25533dcb3a4'
```

Comment:
Prior to merging, we'll need to create a `v1.47.0` in `gutenberg-mobile`, reference that here, and install the pods.